tests/datatypes/test_frozenset.py

```diff
@@ -37,6 +37,60 @@ def test_creation(self):
 
             """)
 
+    def test_contains(self):
+        # Normal Contains
+        self.assertCodeExecution("""
+            x = frozenset()
+            print(x)
```

Comment:
I'm not sure what this is testing, the repr should be tested elsewhere. Should this be ensuring that anything `in x` will be false?